### ISSUE TITLE
Repository

### ISSUE THREAD

**Issue by rkluwen** [domoticz/domoticz#686]
What happened to the Github repository?
First, I needed to re-clone my complete sandbox, because it gave conflicts. Both on the pi and on Windows (since a while already).

Next, I cannot browse my domoticz because of 404 errors:

![image](https://cloud.githubusercontent.com/assets/5295912/15801014/baf5092e-2a88-11e6-96c3-aa720619fe28.png)

I'm doing something terribly wrong here. But what's the issue?

**Comment by gizmocuz** [comment]
Nothing happened?

**Comment by jrich523** [comment]
I've noticed that the msbuild setup for this isnt ideal. Im working (learning) on how to make this a bit more dynamic, but, when it builds there are content folders (www,json,config, scripts, maybe more) that are in the project folder that are not part of the build process. So when you build it, you cant debug it because those items arent in the build folder. 

Based on this it looks like you're having a similar problem since its not finding any of those content folders.

Basically if you want to compile this and be able to run/debug it, or to 'deploy' it, you'll have to modify the project a lot.

**Comment by gerritv** [comment]
@jrich523 To debug under VS you need to add -wwwroot, -dbase and -userdata directives to the Project/properties/debugging/Command arguments property. Point those to the ones in your project structure so that you have a coherent context.

**Comment by gizmocuz** [comment]
or set your startup path to .. (two points, means back one folder)
